src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPageUserControlBase.vb

```diff
@@ -2833,7 +2835,9 @@ NextControl:
 
             'First see if it is already in the project
             For Each ProjectItem As EnvDTE.ProjectItem In ProjectItems
-                If ProjectItem.FileNames(1).Equals(FileName, StringComparison.OrdinalIgnoreCase) Then
+#Disable Warning BC42025 ' Access of shared member, constant member, enum member or nested type through an instance
+                If ProjectItem.get_FileNames(1).Equals(FileName, StringComparison.OrdinalIgnoreCase) Then
+#Enable Warning BC42025 ' Access of shared member, constant member, enum member or nested type through an instance
```

Comment:
These warning wrappers should be removed after #7235 is integrated. Tracked by #7257